chapter_linear-regression/linear-regression-scratch.md

```diff
@@ -89,6 +99,19 @@ class LinearRegressionScratch(d2l.Module):  #@save
             self.b = tf.Variable(b, trainable=True)
 ```
 
+```{.python .input}
+%%tab jax
+class LinearRegressionScratch(d2l.Module):  # @save
+    num_inputs: int
+    lr: float
+    sigma: float = 0.01
```

Comment:
Is this the first occurrence of such practices? If yes, can you add more explanations in text?

Reply:
To be more beginner-friendly, can you describe it briefly in OOD section?